src/network/index.ts

```diff
@@ -1 +1,73 @@
 /** @format */
+import { isEmpty } from "lodash";
+
+import * as Models from "./models";
+import { addHTTPHeaders, addQueryStringParams, buildApiUrl } from "./utils";
+
+async function executeRequest(method: Models.HTTPMethods, url: string, queryStringParameters: Models.IQueryParams = {}, options: RequestInit = {}, payload?: any): Promise<any> {
```

Comment:
param comments?

Reply:
I dont think need to add comments to the params as this function is not accessible outside of the lib itself.